inference-engine/ie_bridges/java/tests/README.md

```diff
@@ -0,0 +1,31 @@
+## Build
```

Comment:
can we extend with different OSes procedures?  or just note that this is Unix only